src/main/java/org/cascadebot/cascadebot/data/objects/GuildPermissions.java

```diff
@@ -147,7 +151,7 @@ public Group createGroup(String name) {
         Group group;
         int iterations = 0;
         do {
-            group = new Group(name);
+            group = new Group(name); //TODO maybe not allow groups with the same name
```

Comment:
> //TODO maybe not allow groups with the same name
Did we talk about this in a meeting? I can't remember

Reply:
we did talk about this i just forgot to remove the todo